integration/query_compat_test.go

```diff
@@ -192,6 +196,122 @@ func testQueryCompat(t *testing.T, testCases map[string]queryCompatTestCase) {
 	testQueryCompatWithProviders(t, shareddata.AllProviders(), testCases)
 }
 
+func TestQueryCappedCollectionCompat(t *testing.T) {
+	t.Skip("https://github.com/FerretDB/FerretDB/issues/3467")
```

Comment:
Skip or delete, haven't run this test yet.

Reply:
It was merged

Reply:
removed skip :+1: